src/python_routing_v01/compute_nhd_routing_SingleSeg.py

```diff
@@ -743,18 +743,18 @@ def compute_level_pool_reach_up2down(
 
     volumec = dt * (quc - qdc + qlat)
     # TODO: This qlatCum is invalid as a cumulative value unless time is factored in
-    qlatCum = qlat * dt
+    qlatCum = qlat
```

Comment:
We need to preserve line 746. 
``` qlatCum = qlat * dt ```

the  `* dt` is important.

Reply:
I will try to put in those changes. This should have been changed in one of my last commits. I will double-check why it did not update.